sys/tests/utest.c

```diff
@@ -60,6 +59,13 @@ static int utest_generic(const char *name, int status_success) {
 #define UTEST_ADD_SIGNAL(name, sig)                                            \
   UTEST_ADD(name, MAKE_STATUS_SIG_TERM(sig), 0)
 
+UTEST_ADD_SIGNAL(vmmap_text_w, SIGSEGV);
+UTEST_ADD_SIGNAL(vmmap_rodata_w, SIGSEGV);
+#ifndef __mips__
+UTEST_ADD_SIGNAL(vmmap_data_x, SIGSEGV);
+UTEST_ADD_SIGNAL(vmmap_rodata_x, SIGSEGV);
+#endif
```

Comment:
I have no idea why these tests still don't work on mips.

Reply:
Actually only `vmmap_rodata_x` does not work ;-)

Reply:
OK. I was wrong. It just fails in another place.

We're going to need `siginfo_t` filled from the kernel to make sure our tests do what they really should check.